Mathlib/Geometry/Manifold/SmoothManifoldWithCorners.lean

```diff
@@ -51,13 +51,13 @@ but add these assumptions later as needed. (Quite a few results still do not req
   we register them as `PartialEquiv`s.
   `extChartAt I x` is the canonical such partial equiv around `x`.
 
-As specific examples of models with corners, we define (in the file `real_instances.lean`)
+As specific examples of models with corners, we define (in the file `Instances/Real.lean`)
```

Comment:
```suggestion
As specific examples of models with corners, we define (in `Geometry.Manifold.Instances.Real`)
```
so that it gets linkified

Reply:
Thanks; done. Two question: is a leading `Mathlib` required? And does `doc-gen` need the `.lean` extension?

Reply:
Leading `Mathlib` I have no clue. Trailing `.lean` is only for filepaths, not module names, so it doesn't apply here.